style.css

```diff
@@ -90,199 +94,355 @@ header > .logo {
   color: #344563;
 }
 
+.social-anchors {
+  display: flex;
+}
+
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 10px 16px;
+  margin: 10px auto;
+  width: 327px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
```

Comment:
- [Optional] It would be nice if you could increase the vertical margin between works section cards so that your design resembles the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) template as much as possible . Cheers 👍 

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image (2)](https://user-images.githubusercontent.com/60618877/178112087-fed5eca5-a876-4bb4-abd6-540e2a0edd37.png)  | ![image (1)](https://user-images.githubusercontent.com/60618877/178112050-9674ff8b-9c94-4bee-ae5d-8ec1a37f269a.png)